crates/ef-testing/src/evm_sequencer/constants.rs

```diff
@@ -41,9 +41,9 @@ lazy_static! {
             block_number: BlockNumber(0),
             block_timestamp: BlockTimestamp(0),
             sequencer_address: *SEQUENCER_ADDRESS,
-            fee_token_address: *FEE_TOKEN_ADDRESS,
+            fee_token_addresses: FeeTokenAddresses{eth_fee_token_address: *ETH_FEE_TOKEN_ADDRESS, strk_fee_token_address: *STRK_FEE_TOKEN_ADDRESS},
             vm_resource_fee_cost: Arc::new(VM_RESOURCES.clone()),
-            gas_price: 1,
+            gas_prices: GasPrices{eth_l1_gas_price: 1, strk_l1_gas_price: 1},
```

Comment:
can you add comment explaining what this does?

Reply:
Same here, blockifier update